src/js/main.js

```diff
@@ -1,10 +1,15 @@
 "use strict";
 
-var React = require("react");
-var App = require("./app");
-var API = require("./api.js");
-var baseUrl = process.env.READINGLIST_SERVER_BASEURL || "http://0.0.0.0:8000/v0";
+import React from "react/addons";
+import API from "./api";
+import App from "./app";
+import {ArticleStore, stores} from "./flux";
 
-var api = new API(baseUrl);
+// XXX handle this with envify
+var serverUrl = process.env.READINGLIST_SERVER_BASEURL || "http://0.0.0.0:8000/v0";
```

Comment:
This `process.env` thing is provided by browserify, to emulate a real nodejs environment. As mentioned previously, we'll use envify to transform these at compilation time.